frontend/src/scenes/error-tracking/ErrorTrackingFilters.tsx

```diff
@@ -18,12 +18,23 @@ export const FilterGroup = (): JSX.Element => {
 
     return (
         <div className="flex flex-1 items-center justify-between space-x-2">
-            <div className="flex items-center gap-2">
-                <LemonInput type="search" placeholder="Search..." value={searchQuery} onChange={setSearchQuery} />
+            <div className="flex flex-1 items-center gap-2 mx-2">
+                <LemonInput
+                    type="search"
+                    placeholder="Search..."
+                    value={searchQuery}
+                    onChange={setSearchQuery}
+                    className="flex-grow max-w-none"
+                />
                 <UniversalFilters
                     rootKey="error-tracking"
                     group={filterGroup}
-                    taxonomicGroupTypes={[TaxonomicFilterGroupType.PersonProperties, TaxonomicFilterGroupType.Cohorts]}
+                    // TODO: Probably makes sense to create a new taxonomic group for exception-specific event property filters only, keep it clean.
```

Comment:
I guess as long as we rely on super properties to track "source_repo" (or whatever a customer wants to call their property) we need to give them a way to select any property. Unless we let them pick a property they're going to use as the repository source and save that somehow

Reply:
yeah, I'm not yet sure if we want to give full freedom, and if we do, makes sense to sort this list by more relevant properties on top.

Reply:
We could continue recommending `super_properties` but tell people to use `$source_repo` as the specific property